pyanaconda/addons.py

```diff
@@ -27,7 +27,7 @@
 from pyanaconda.progress import progress_message
 from pyanaconda.i18n import N_
 
-log = logging.getLogger("anaconda")
+log = logging.getLogger("anaconda.addons")
```

Comment:
I thought about just basing the prefix on folder structure (which I think is basically what the module property contains), but then decided against that, for a couple reasons:
- the prefix would change if we move the file around, making it less stable
- manually specifying the prefix is simple (a single line per module) and provides greater flexibility
  - related modules can be grouped together with a single prefix without needlessly exposing filesystem details (eq. all non spoke & hub modules can just live in the ui.gui.* namespace while spokes log to ui.gui.spoke.* and hubs log to ui.gui.hub.*)
  - as the prefix is not based on folder structure a more meaningful & better readable prefix can be used
    - `ui.gui.spoke.network` instead of `ui.gui.spokes.network` - the log messages come from **the** network spoke, not one of the network spokes
    - `ui.gui.spoke.custom_storage_config` instead of `ui.gui.spokes.custom` - even though anaconda developers, contributors and the QA community probably know that "custom spoke" is actually the "custom storage configuration spoke", not everyone might know that, especially when reading a log message prefix
- it should be pretty easy to track message origin by grepping for the prefix in the Anaconda source code (the module based approach would have a similar benefit though)
- there might actually be more than one logger in a file, if the functionality contained in it is disparate enough (as for example in the kickstart module)

In short - I want the log prefix to understandably describe what is being logged and to be reasonably stable over time.